### PR TITLE
Mm/track preview

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
Adds two tracking events to slice-machihne.

It was going to be three but the iframe will needed to be modified to handle these.

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx [hunk 1]
```diff
@@ -171,7 +172,7 @@ const StoryBookOrPreview: React.FC<{
           mt: 3,
           color: "textGray",
           "::first-letter": {
-            "text-transform": "uppercase",
+            textTransform: "uppercase",
```
Comment (Miniplop) [inline packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx:175]: good catch

File: packages/slice-machine/src/modules/environment/index.ts [hunk 2]
```diff
@@ -49,8 +49,19 @@ export const getUpdateVersionInfo = (
...
+};
+
+export const getStorybookUrl = (
+  store: SliceMachineStoreType
+): string | null => {
+  return store.environment.env.manifest.storybook || null;
+};
+
+export const getUserID = (store: SliceMachineStoreType): string | null => {
+  return store.environment.env.userId || null;
```
Comment (Miniplop) [inline packages/slice-machine/src/modules/environment/index.ts]: I think you can return 'store.environment.env.userId' directly as this is already null | string :)

**Review by Miniplop — COMMENTED** [review]
I think you can remove all the logic to send the userID to segment, because this will be change when we will make the reconciliation on login. 

I let you also some other comments :)

File: packages/slice-machine/src/modules/environment/index.ts (hunk 2 above)
Comment (Miniplop) [reply packages/slice-machine/src/modules/environment/index.ts]: Ignore this comment because you will remove this selector :)

Comment (MarcMcIntosh) [reply packages/slice-machine/src/modules/environment/index.ts]: done

**Comment by Miniplop** [comment]
LGTM

**Review by Miniplop — APPROVED** [review]

**Pull request merged**